bears/python/PycodestyleBear.py

```diff
@@ -37,8 +37,10 @@ def create_arguments(
         """
         arguments = [r'--format=%(row)d %(col)d %(code)s %(text)s']
 
+        ignore_values = 'E121, E123, E126, E133, E226, E241, E242, E704, W503'
```

Comment:
makes those rather the default above:
```
pycodestyle_ignore: str='E121, E123, ...'
```
then you don't have to change any code and the user can enable those implicitly disabled defaults again. Otherwise we would restrict people :+1: